core/src/test/java/org/elasticsearch/index/query/RangeQueryBuilderTest.java

```diff
@@ -82,6 +92,13 @@ protected RangeQueryBuilder doCreateTestQueryBuilder() {
         return query;
     }
 
+    public RangeQueryBuilder createRandomStringRangeQuery() {
```

Comment:
Could be package private, since I want to access it from RandomQueryBuilder, but for same reasong protected wouldn't work.